package.nls.json

```diff
@@ -149,6 +149,7 @@
     "AWS.command.samcli.sync": "Sync SAM Application",
     "AWS.command.samcli.syncCode": "Sync SAM Application (code only)",
     "AWS.command.s3.downloadFileAs": "Download As...",
+    "AWS.command.s3.downloadFolder": "Download Folder...",
```

Comment:
is a separate command needed or could we use `"Download As..."` (and `aws.s3.downloadFileAs`) on folder paths?